### PR TITLE
(DOCSP-22157): Endpoint Return Types

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- https://jira.mongodb.org/browse/DOCSP-22157

### Staged Changes (Requires MongoDB Corp SSO)

- [Endpoints - Return Types](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-22157/endpoints/#return-types)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/endpoints.txt [hunk 1]
```diff
@@ -270,6 +270,76 @@ includes methods that you can call to configure the response.
...
+By default, endpoints encode response body content in JSON.
+MongoDB stores data as BSON and uses :manual:`MongoDB Extended JSON
+</reference/mongodb-extended-json/>`, or EJSON, to
+represent types that do not exist in the JSON format.
+To translate MongoDB EJSON data into JSON responses, endpoints
+translate fields into JSON values when possible and
+omit the values of EJSON fields that have no JSON equivalent.
+
+To avoid any field value translations or omissions, configure
+your endpoint to respond with EJSON. Use the ``Accept``
```
Comment (jo-tse-mongo) [inline source/endpoints.txt:287]: I'm wondering if the Return Type configuration should be first and then the Accept header. 

The way I imagined it is that the Accept header is for certain cases where you would want to have a different response than what's stated in the Return Type for an Endpoint. The first thing users should do is configure their Endpoints to return what they want most of the time, and they can use the Accept header sometimes

**Review by jo-tse-mongo — APPROVED** [review]
LGTM - I have a question about the ordering of the Accept Header and Return Type sections

File: source/endpoints.txt (hunk 1 above)
Comment (nathan-contino-mongo) [reply source/endpoints.txt]: Updated the ordering, this is a great point.

File: source/endpoints.txt [hunk 2]
```diff
@@ -270,6 +270,85 @@ includes methods that you can call to configure the response.
...
+             "Name": "Mango",
+             "Year": { "$numberLong": "2022" },
+             "Weight": { "$numberDecimal": "9823.1297" },
+             "Date": { "$date": { "$numberLong": "1641954803067" } }
+          }
+
+.. seealso::
+
+   For more information about the differences between JSON and EJSON
+   responses, see the :atlas:`Atlas Data API documentation <api/data-api/#extended-json-responses>`.
```
Comment (nathan-contino-mongo) [inline source/endpoints.txt:350]:
```suggestion
   responses, see the :atlas:`Atlas Data API documentation </api/data-api/#extended-json-responses>`.
```

**Comment by nathan-contino-mongo** [comment]
Already reviewed this content in cloud-docs, merging in so we have coverage of this now-released feature.

**Pull request merged**